### PR TITLE
4.2.0 MOL-356  1.7.7 second chance email fix

### PR DESCRIPTION
**Pull request by lukvai**

### CODE REVIEW

File: controllers/front/return.php [hunk 1]
```diff
@@ -214,27 +214,27 @@ protected function processGetStatus()
...
 		/** @var PaymentMethodRepository $paymentMethodRepo */
-		$paymentMethodRepo = $this->module->getContainer(PaymentMethodRepository::class);
+		$paymentMethodRepo = $this->module->getMollieContainer(PaymentMethodRepository::class);
 
 		$transactionId = Tools::getValue('transaction_id');
 		$dbPayment = $paymentMethodRepo->getPaymentBy('transaction_id', $transactionId);
 		$cart = new Cart($dbPayment['cart_id']);
 		if (!Validate::isLoadedObject($cart)) {
-			die(json_encode([
+			exit(json_encode([
```
Comment (tomas862) [inline controllers/front/return.php:223]: whats the argument for this change ?

File: src/Grid/Action/Type/SecondChanceRowAction.php [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+			->setAllowedTypes('route_param_field', 'string')
+			->setAllowedTypes('accessibility_checker', [AccessibilityCheckerInterface::class, 'callable', 'null'])
+			->setAllowedTypes('use_inline_display', 'bool')
+		;
+	}
+
+	/**
+	 * {@inheritdoc}
+	 */
+	public function isApplicable(array $record)
```
Comment (tomas862) [inline src/Grid/Action/Type/SecondChanceRowAction.php:74]: do you use this ?

File: controllers/front/return.php (hunk 1 above)
Comment (tomjas1997) [reply controllers/front/return.php:223]: Unsure why merging changed this, will fix it in other places aswell.

Comment (tomas862) [reply controllers/front/return.php:223]: maybe its cs fixer rule or something then  - anyway, need to understand then why its better to use this rather then die.

**Review by tomas862 — APPROVED** [review]

**Pull request merged**